docs/core_docs/.gitignore

```diff
@@ -43,6 +43,8 @@ docs/use_cases/question_answering/sources.md
 docs/use_cases/question_answering/sources.mdx
```

Comment:
seems to have happened automatically - didn't manually change this

Reply:
Yeah, all good there